### ISSUE TITLE
Add Tooltips to Meta Data on Book View Page

### ISSUE THREAD

**Issue by mendahu** [mendahu/off-nominal-book-club#5]
Read, Wishlist, Favourites should have hover tool tips for accessibility and new user teaching
